logical/order_by.go

```diff
@@ -56,5 +56,8 @@ func (node *OrderBy) Physical(ctx context.Context, physicalCreator *PhysicalPlan
 		}
 	}
 
-	return physical.NewOrderBy(expressions, directions, sourceNode), variables, nil
+	// OrderBy operates on a single, joined stream.
```

Comment:
What do you mean by joined? (it also applies to the previous "... operates on a single, joined stream" comments). Does it mean that OrderBy functions the same on a single stream as on the joining of a partitioned stream?

Reply:
Yes, it first joins all partitions into one partition using a shuffle (for now)